website/docs/docs/dbt-versions/2023-release-notes.md

```diff
@@ -0,0 +1,998 @@
+
+    To learn more, refer to [Extended attributes](/docs/dbt-cloud-environments#extended-attributes).
+
+    The **Extended Atrributes** text box is available from your environment's settings page: 
+
+    <Lightbox src="/img/docs/dbt-cloud/using-dbt-cloud/extended-attributes.jpg" width="85%" title="Example of the Extended Attributes text box" />
+
+  </expandable>
+
+- <expandable alt_header='Legacy semantic layer'>
+
+    dbt Labs has deprecated dbt Metrics and the legacy dbt Semantic Layer, both supported on dbt version 1.5 or lower. This change starts on December 15th, 2023.
+
+    This deprecation means dbt Metrics and the legacy Semantic Layer are no longer supported. We also removed the feature from the dbt Cloud user interface and documentation site.
+
+    ### Why this change?
+
+    The [re-released dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl), powered by MetricFlow, offers enhanced flexibility, performance, and user experience, marking a significant advancement for the dbt community.
+
+    ### Key changes and impact
+
+    - **Deprecation date** &mdash; The legacy Semantic Layer and dbt Metrics will be officially deprecated on December 15th, 2023.
+    - **Replacement** &mdash; [MetricFlow](/docs/build/build-metrics-intro) replaces dbt Metrics for defining semantic logic. The `dbt_metrics` package will no longer be supported post-deprecation.
+    - **New feature** &mdash; Exports replaces the materializing data with `metrics.calculate` functionality and will be available in dbt Cloud in December or January.
+
+    ### Breaking changes and recommendations
+
+    - For users on dbt version 1.5 and lower with dbt Metrics and Snowflake proxy:
+    - **Impact**: Post-deprecation, queries using the proxy _will not_ run.
+    - **Action required:** _Immediate_ migration is necessary. Refer to the [dbt Semantic Layer migration guide](/guides/sl-migration?step=1)
+
+    - For users on dbt version 1.5 and lower using dbt Metrics without Snowflake proxy:
+    - **Impact**: No immediate disruption, but the package will not receive updates or support after deprecation
+    - **Recommendation**: Plan migration to the re-released Semantic Layer for compatibility with dbt version 1.6 and higher.
+
+    ### Engage and support
+
+    - Feedback and community support &mdash; Engage and share feedback with the dbt Labs team and dbt Community slack using channels like [#dbt-cloud-semantic-layer](https://getdbt.slack.com/archives/C046L0VTVR6) and [#dbt-metricflow](https://getdbt.slack.com/archives/C02CCBBBR1D). Or reach out to your dbt Cloud account representative.
+    - Resources for upgrading &mdash; Refer to some additional info and resources to help you upgrade your dbt version:
+    - [Upgrade version in dbt Cloud](/docs/dbt-versions/upgrade-dbt-version-in-cloud)
+    - [Version migration guides](/docs/dbt-versions/core-upgrade)
+
+  </expandable>
+
+## November 2023
+
+- <expandable alt_header='dbt Explorer updates'>
```

Comment:
this includes job notification updates and not the explorer items as the title suggested. i think this is the content  [pr here](https://github.com/dbt-labs/docs.getdbt.com/pull/5510/files#diff-4821b108fa35c74fbb7eedfcf49008af8e8453bf1d8f06d73f9f7cb7700e9157L1-L6):



dbt Labs is excited to announce the latest features and UI updates to dbt Explorer!

For more details, refer to [Explore your dbt projects](/docs/collaborate/explore-projects).

## The project's lineage graph 

- The search bar in the full lineage graph is now more prominent. 
- It's easier to navigate across projects using the breadcrumbs.
- The new context menu (right click) makes it easier to focus on a node or to view its lineage.

<LoomVideo id='1d28cc6443a94f00bbcb43d6ab3ad0d9' />

## Search improvements 

- When searching with keywords, a new side panel UI helps you filter search results by resource type, tag, column, and other key properties (instead of manually defining selectors). 
- Search result logic is clearly explained. For instance, indicating whether a resource contains a column name (exact match only).

<LoomVideo id='5c4dc30afa824897a8c8b3ed37cee782?t=14' />

## Resource details
- Model test result statuses are now displayed on the model details page. 
- Column names can now be searched within the list.

<LoomVideo id='04f3e35cdc0c41109e91f7692d432059' />